### PR TITLE
KAZOO-4413 upgrade poolboy to 1.5.1

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

File: deps/poolboy-1.5.1/rebar.config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{erl_opts, [
+  debug_info,
+  {platform_define, "^R", pre17}
```
Comment (fenollp) [inline deps/poolboy-1.5.1/rebar.config]: Does this mean we can't use it with OTP ≥ 17?

File: deps/poolboy-1.5.1/Makefile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+REBAR = $(shell command -v rebar || echo ./rebar)
```
Comment (fenollp) [inline deps/poolboy-1.5.1/Makefile]: Use the rebar bin we already have in /utils

Comment (fenollp) [inline deps/poolboy-1.5.1/Makefile]: Same for #1468 : remove the rebar binaries

Comment (lazedo) [inline deps/poolboy-1.5.1/Makefile]: @fenollp rebar came with the repo, i'll remove it if you update our rebar in utils which is quite old :)

Comment (fenollp) [inline deps/poolboy-1.5.1/Makefile]: I'm fine with that

**Pull request merged**